packages/flutter/lib/src/widgets/form.dart

```diff
@@ -125,6 +136,16 @@ class Form extends StatefulWidget {
   /// will rebuild.
   final VoidCallback onChanged;
 
+  /// If [AutoValidateMode.onUserInteraction] this form will only auto-validate
+  /// after its content changes and if [AutoValidateMode.always] will
+  /// autovalidate without user interaction.
+  ///
+  /// Defaults to [AutoValidateMode.disabled] which means no auto-validation
+  /// will occur.
+  ///
+  /// This property has no effect if [autovalidate] is false.
```

Comment:
This seems incorrect?

Reply:
Yes. I need to remove this statement:

` /// This property has no effect if [autovalidate] is false.`